### ISSUE TITLE
`html-indent` reports false positive.

### ISSUE THREAD

**Issue by ota-meshi** [vuejs/eslint-plugin-vue#729]
**Tell us about your environment**	
* **ESLint version:**	5.10.0
* **eslint-plugin-vue version:**	5.0.0
* **Node version:**	8.11.3

**Please show your full configuration:**

```js
module.exports = {
  rules: {
    "vue/html-indent": ["error"]
  }
```

**What did you do?**	

```vue
<template>
  <div>text
    <a/>
  </div>
  <div>text
    text
    <a/>
  </div>
  <div><a/>
    <a/>
  </div>

  <!-- when there is a line break before the first content -->
  <div>
    <a/>
    <a/>
  </div>
  <div>
    text
    <a/>
  </div>
  <div>
    text
    text
    <a/>
  </div>
</template>
```

**What did you expect to happen?**
I think there is no errors.

**What actually happened?**

If there is no line break before the first content, false positives are reported.

```
Expected indentation of 2 spaces but found 4 spaces. [vue/html-indent] (3, 1)
Expected indentation of 2 spaces but found 4 spaces. [vue/html-indent] (6, 1)
Expected indentation of 2 spaces but found 4 spaces. [vue/html-indent] (7, 1)
Expected indentation of 7 spaces but found 4 spaces. [vue/html-indent] (10, 1)
```

![image](https://user-images.githubusercontent.com/16508807/50128498-86c7f480-02b8-11e9-8419-cf471bc6221f.png)

fixed:

![image](https://user-images.githubusercontent.com/16508807/50128527-aeb75800-02b8-11e9-8760-48afe61feac2.png)
